Documentation/Rendering/ClippingPrimitive.md

```diff
@@ -0,0 +1,114 @@
+# Clipping Primitive
+
+The [`ClippingPrimitive`](xref:Microsoft.MixedReality.Toolkit.Utilities.ClippingPrimitive) behaviors allow for performant [`plane`](xref:Microsoft.MixedReality.Toolkit.Utilities.ClippingPlane), [`sphere`](xref:Microsoft.MixedReality.Toolkit.Utilities.ClippingSphere), and [`box`](xref:Microsoft.MixedReality.Toolkit.Utilities.ClippingBox) shape clipping with the ability to specify which side of the primitive to clip against (inside or outside) when used with MRTK shaders.
+
+![primitive clipping gizmos](../../Documentation/Images/MRTKStandardShader/MRTK_PrimitiveClippingGizmos.gif)
```

Comment:
nit: can you remove ../../documentation and simplify to shorter path

Reply:
Nice catch!